pyacq/devices/npbufferdevice.py

```diff
@@ -40,14 +40,12 @@ def _configure(self, nb_channel=16, sample_interval=0.001, chunksize=256,
         self.chunksize = chunksize
         self.timeaxis = timeaxis
         
-        
+        self.output.spec['shape'] = (-1, nb_channel)
         if self.timeaxis == 0:
-            self.output.spec['shape'] = (-1, nb_channel)
-            self.output.spec['timeaxis'] = self.timeaxis
+            self.output.spec['axisorder'] = None
             self.channelaxis = 1
         else:
-            self.output.spec['shape'] = (nb_channel, -1)
```

Comment:
If you don't mind, I would prefer to fix this stream in the master.
And to come back on others PR later on.
The PR #50 include other changes and this PR is big enougth no ?
I also would like to be sure that the new design in buffer in #50 also enable to swapaxis internally. This is important for testing. No ?

The merge will be more work but we will be able to keep only the PR changes for npbufferdevice.py